lib/common/tenant.js

```diff
@@ -64,7 +64,8 @@ function remove (tenantName) {
 }
 
 module.exports = {
-  get,
-  set,
+  // Can not use same syntax as remove below to support Node 4.
+  get: get,
+  set: set,
```

Comment:
This is interesting. I thought Node 4.x supported object shorthand properties. At least, it does according to http://node.green/

It's also super weird that `{ get: get }` is broken, but `{ remove }` is not. Perhaps this is more to do with shorthand getter|setter methods than it is to do with shorthand properties? /shrug